graphbolt/src/cuda/extension/gpu_graph_cache.cu

```diff
@@ -510,12 +506,10 @@ std::tuple<torch::Tensor, std::vector<torch::Tensor>> GpuGraphCache::Replace(
               }
               if (edge_id_offsets) {
                 // Append the edge ids as the last element of the output.
-                output_edge_tensors.push_back(
-                    ops::IndptrEdgeIdsImpl(
-                        output_indptr, output_indptr.scalar_type(),
-                        *edge_id_offsets,
-                        static_cast<int64_t>(
-                            static_cast<indptr_t>(output_size))));
+                output_edge_tensors.push_back(ops::IndptrEdgeIdsImpl(
+                    output_indptr, output_indptr.scalar_type(),
+                    *edge_id_offsets,
+                    static_cast<int64_t>(static_cast<indptr_t>(output_size))));
```

Comment:
Is this just a reformatting? Maybe revert, as it seems unrelated?

Reply:
Ah, I think such linters should really only be looking at modified lines, but I see it comes from upstream, so no need to fight it.